src/main/java/org/folio/gobi/HelperUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.gobi;
+
+import java.util.concurrent.CompletionException;
+
+import org.folio.gobi.exceptions.HttpException;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class HelperUtils {
+
+  private HelperUtils() {
+
+  }
+
+  public static String truncate(String message, int limit) {
+    return message.substring(0, Math.min(message.length(), limit));
+  }
+
+  public static JsonObject verifyAndExtractBody(org.folio.rest.tools.client.Response response) {
+    if (!org.folio.rest.tools.client.Response.isSuccess(response.getCode())) {
```

Comment:
I'm not positive I know what you mean...  Typically this is invoked asynchronously immediately after an httpClient request...   If you're wondering if there's a missing null check, I suppose it couldn't hurt to add one, but typically if something went wrong during the request, it would be handled via exception handling and this method wouldn't be called.

Reply:
A null check makes sense if there's a possibility that it can be null, but if the Response object is created by the framework and is always not null with all of its properties populated ( at least the ones that we care about), then there's no need to have a null check.